test/activities.js

```diff
@@ -0,0 +1,151 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const server = "http://localhost:3000";
+const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjI1LCJyb2xlSWQiOjEsImlhdCI6MTY1MDkyMjIyNCwiZXhwIjoxNjUxMDA4NjI0fQ.cXzKVKxt2DXPkfy9KTjSBcuBLDH8MktEEhF1bYi2YpM"
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Test Activities endpoint', () => {
+  describe('POST /activities', () => {
+    it('It should POST a new activities', (done) => {
```

Comment:
Fijate que la función se llama it, por ésto mismo, no hace falta agregar el It al texto, generalmente arranca con should. Lo mismo para el resto